UiTests/Common/UiTestHelpers.cs

```diff
@@ -0,0 +1,572 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using Azure.Security.KeyVault.Secrets;
+using Microsoft.Playwright;
+using System.Diagnostics;
+using System.Management;
+using System.Runtime.Versioning;
+using System.Text;
+using Xunit.Abstractions;
+
+namespace Common
+{
+    public static class UiTestHelpers
+    {
+        /// <summary>
+        /// Navigates to a web page with retry logic to ensure establish a connection in case a web app needs more startup time.
+        /// </summary>
+        /// <param name="uri">The uri to navigate to</param>
+        /// <param name="page">A page in a playwright browser</param>
+        /// <returns></returns>
```

Comment:
Can this be removed?

Reply:
The returns line can be, removed "returns" tags.